index.js

```diff
@@ -15,7 +24,8 @@ exports.parse = function parse (line, startAt) {
         strict: false, // turn off obs- features in the rfc
         rejectTLD: false, // domains require a "."
         startAt: startAt || null,
-        atInDisplayName: true // allow at in display name
+        atInDisplayName: true, // allow at in display name
```

Comment:
Now that we have a method for setting options, how about also making `atInDisplayName` configurable?

Reply:
Sure, will add it!